src/sagemaker/feature_store/feature_group.py

```diff
@@ -607,6 +616,7 @@ def ingest(
             wait (bool): whether to wait for the ingestion to finish or not.
             timeout (Union[int, float]): ``concurrent.futures.TimeoutError`` will be raised
                 if timeout is reached.
+            profile_name (str): the profile credential should be used for ``PutRecord``.
```

Comment:
Let's add more detail here. It should include 1/ what the default behavior is, if no profile name is specified and 2/ clarify that the profile credential is provided to a new boto3 Session, which is used initialize the sagemaker_featurestore_runtime client to ingest the data.